src/nprintml/learn/automl.py

```diff
@@ -57,11 +57,9 @@ class AutoML:
         'best_quality',
     )
 
-    N_THREADS = None
```

Comment:
It appears that this thread control has been completely removed from the tabular API. I can't say much about it, but regardless, we can just forget about it here, for now.

Reply:
This seems really bad, does it still use multiple threads when actually running?

Reply:
I'm having trouble tracking down what changed, but it _seems_ totally fine.

They definitely removed the flag. But [in this example output it is automagically threading](https://auto.gluon.ai/dev/tutorials/tabular_prediction/tabular-multimodal-text-others.html#autogluon-tabular-with-multimodal-support) as it sees fit.